src/vm/assembly.cpp

```diff
@@ -179,6 +183,10 @@ void Assembly::Init(AllocMemTracker *pamTracker, LoaderAllocator *pLoaderAllocat
 #endif
         m_pManifest = Module::Create(this, mdFileNil, GetManifestFile(), pamTracker);
 
+#ifdef FEATURE_EVENT_TRACE
+    g_cAssemblies++;
```

Comment:
Should this use Interlocked operations to make this reliable? Otherwise, the error can accumulate over time - when there is a lot loading/unloading.